libs/agora/gene-comparison-tool/src/lib/gene-comparison-tool.component.ts

```diff
@@ -85,21 +87,28 @@ import { GeneComparisonToolLegendPanelComponent } from './components/gene-compar
 })
 export class GeneComparisonToolComponent implements OnInit, AfterViewInit, OnDestroy {
   router = inject(Router);
-  route = inject(ActivatedRoute);
   geneService = inject(GenesService);
   distributionService = inject(DistributionService);
   helperService = inject(HelperService);
   messageService = inject(MessageService);
   filterService = inject(FilterService);
 
+  /* Query Parameters ------------------------------------------------------ */
+  @Input('category') categoryParam = '';
+  @Input('subCategory') subCategoryParam = '';
+  @Input('sortField') sortFieldParam = '';
+  @Input('sortOrder') sortOrderParam = '';
+  @Input('significance') significanceParam = '';
```

Comment:
My own convention I've come up with is to add 'Param' as a suffix.  One could use it without the aliasing.  However, many of these variables (like category and subCategory already exist, are modified after initialization and are heavily used in the code).  For clarity sake, adding this aliasing should make the code a little easier to read.